multiaccounts/settings/database_settings_manager.go

```diff
@@ -0,0 +1,77 @@
+package settings
```

Comment:
This interface should have been split to make the code more easily testable.
Code smell can be seen when seeing in go struct some 

```
*settings.Database
```
As this means we are forcing the developer/tester to pass an actual implementation of the Database. This is really sub-optimal because it can take some time to setup all the parameters required to run the application.
Instead, this PR proposal is to use an interface and be able to mock the critical function that are being called. 
With this PR we do not go level to call the database, but we mock the call hence we control the result from the low level API call.
As a result, we can avoid flaky tests, and have fast tests.